src/Raven.Server/Indexing/LuceneVoronDirectory.cs

```diff
@@ -68,19 +68,18 @@ public override void TouchFile(string name)
             if (readResult == null)
                 throw new FileNotFoundException("Could not find file", name);
 
-            filesTree.Add(name, Stream.Null);
+            var pos = filesTree.DirectAdd(name, readResult.Reader.Length);
```

Comment:
What is the point of doing it here? We are reading & writing the same value. The end result is a noop.